main.go

```diff
@@ -229,6 +229,10 @@ func serveHTTP(rw http.ResponseWriter, r *http.Request) {
 	}
 
 	switch r.URL.Path {
+	case "/health":
```

Comment:
what is the link with the skip tls feature?

Reply:
Oops, sorry, I meant to put this in the PR description but was rushing out the door and forgot.  It's not linked to the skip_tls feature but was definitely something we found useful internally when running under kubernetes.  I can flesh it out here or open a different PR if you'd prefer.

Reply:
yes, it's better to put it in another PR. FYI we're also using chproxy under kube and we don't need the /health endpoint

Reply:
👍

Reply:
Belated followup question: when you run chproxy under k8s are you using a livenessProbe or a readinessProbe and if so what URL do you use for that?

Reply:
I'll let @sigua-cs answer since I didn't worked on the kube part but I think we're using the /ping

Reply:
@n-oden in our statefulset manifest, we use `/metrics` endpoint for both probes
`          livenessProbe:
            httpGet:
              path: /metrics
              port: http
          readinessProbe:
            httpGet:
              path: /metrics
              port: http`